protocols/virtualfund/virtualfund.go

```diff
@@ -501,7 +498,6 @@ type GetTwoPartyConsensusLedgerFunction func(counterparty types.Address) (ledger
 func ConstructObjectiveFromMessage(
 	m protocols.Message,
 	myAddress types.Address,
-	getTwoPartyLedger GetTwoPartyLedgerFunction,
```

Comment:
Can we be warned about unused parameters through static checks?

Reply:
Not via any well-supported linters. Go allows unused params because particular implementations of some interface might not use all params of a given method.

(bad) example:

```
type 2dShapeUtil interface {
  Area(dim1 float, dim2 float) float
}

type CircleUtil struct {}
func (c CircleUtils) Area(dim1 float, dim2 float) float { return pi*dim1*dim1 }
```